WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -1554,7 +1554,7 @@ function(date, duration, roll) {
         if (ZmId.VIEW_CAL_FB == this._currentViewType && roll && appCtxt.get(ZmSetting.FREE_BUSY_VIEW_ENABLED)) {
             currentView._navDateChangeListener(date);
 		}
-		this._navToolBar[ZmId.VIEW_CAL].setText(title, 'polite');
+		this._navToolBar[ZmId.VIEW_CAL].setText(title);
```

Comment:
FYI. Remove updating `aria-live` attribute with same value `polite` on each next/prev button click call. We already have set `aria-live='polite'` attribute to the calendar navigation text element at time of initialisation.